src/OpenTelemetry.Api/Internal/SemanticConventions.cs

```diff
@@ -110,5 +110,19 @@ internal static class SemanticConventions
         public const string AttributeExceptionType = "exception.type";
         public const string AttributeExceptionMessage = "exception.message";
         public const string AttributeExceptionStacktrace = "exception.stacktrace";
+
+        // NEW v1.21.0 Http Semantic Conventions
+        public const string AttributeClientSocketPort = "client.socket.port";
+        public const string AttributeHttpRequestMethod = "http.request.method";
+        public const string AttributeHttpResponseStatusCode = "http.response.status_code";
+        public const string AttributeNetworkProtocolVersion = "network.protocol.version";
+        public const string AttributeServerAddress = "server.address";
+        public const string AttributeServerName = "server.name";
```

Comment:
where is `server.name` mentioned in spec?

Reply:
good catch!
looks like a typo. net.host.name -> server.address
looks like i did not make the same mistake in the other pr.